docs/user/tutorial/quickstart/java11.rst

```diff
@@ -1,152 +0,0 @@
-:Author: Torben Barsballe
-:Thanks: geotools-user list
-:Version: |release|
-:License: Creative Commons with attribution
-
-**********************
- Java 11 Quickstart 
-**********************
-
-.. sectionauthor:: Torben Barsballe <tbarsballe@boundlessgeo.com>
-
-Welcome 
-=======
-
-GeoTools 21.0 and newer can be used in a Java 11 modular application. This requires a few additions on top of the regular tutorial application.
-
-This quickstart assumes you have already run through one of the regular quickstarts:
-
- * `Eclipse <./eclipse.html>`_
- * `NetBeans <./netbeans.html>`_
- * `IntelliJ <./intellij.html>`_
- * `Maven <./maven.html>`_
-
-Java Install
-============
-
-Instead of Java 8, we will want to install Java 11.
-
-#. Download the latest Java 11 JDK:
-
-   * OpenJDK: http://jdk.java.net/11/
-
-   .. info:
-
-      OpenJDK updates every six months, for longer term support consider:
-      
-      * Adopt OpenJDK: https://adoptopenjdk.net
-      
-      Oracle distributes a supported JDK 11 for their customers, which is available for development and testing. It is **not** available for production use without a license:
-      
-      * http://www.oracle.com/technetwork/java/javase/downloads/
-   
-#. At the time of writing the latest Java 11 release was:
-   
-   * ``jdk-11.0.1``
-   
-#. Click through the installer you will need to set an acceptance a license agreement and so forth.
-   By default this will install to:
-   
-   * :file:`C:\\Program Files\\Java\jdk-11.0.1`
-
-#. Update the ``JAVA_HOME`` system variable so that maven uses the newly installed Java 11:
-
-   * JAVA_HOME = :file:`C:\\Program Files\\Java\\jdk-11.0.1`
-
-#. Verify ``mvn`` is using the correct Java version by running:
-
-   .. code-block:: sh
-
-       mvn -version
-
-   It should report a Java version of 11.0.1.
-
-.. Note::
-
-   In this tutorial we refer to file and directory paths as used by Windows. If you are fortunate
-   enough to be using another operating system such as Linux or OSX all of the commands and source
-   code below will work, just modify the paths to suit. 
-
-Updating the POM
-----------------
-
-A few changes are required to compile and run on Java 11:
-
-#. Add dependencies for **gt-render** and **gt-main**, since we import packages from them:
-
-    .. literalinclude:: artifacts/pom-jdk11.xml
-        :language: xml
-        :start-after: </properties>
-        :end-before: <repositories>
-
-#. Update the Maven plugin versions. We want to ensure we use the most recent, Java 11-compatible, versions. We will also update the configuration of the Maven compile plugin to target Java 11:
-
-    .. literalinclude:: artifacts/pom-jdk11.xml
-        :language: xml
-        :start-after: </repositories>
-        :end-before: </project>
-
-Adding Module Info
```

Comment:
This section should be added to the other quickstart tutorials.

Reply:
Recovered and moved in its own dedicated section (rather than duplicating it all over the place).